models/refugee.py

```diff
@@ -0,0 +1,122 @@
+from datetime import date, datetime
+from enum import Enum
+
+class Refugee:
+    """
+    A class to represent a refugee family.
+    """
+    class MedicalCondition(Enum):
+        CANCER = "Cancer"
+        CHRONICKIDNEY = "Chronic kidney disease"
+        CHRONICLIVER = "Chronic liver disease"
+        CHRONICLUNG = "Chronic lung diseases"
+        DEMENTIA_OR_NEUROLOGICAL = "Dementia or other neurological conditions"
+        DIABETES = "Diabetes"
+        DOWNSYNDROME = "Down Syndrome"
+        HEART = "Heart conditions"
+        HIV = "HIV infection"
+        IMMUNOCOMPROMISED = "Immunocompromised state (weakened immune system)"
+        MENTALHEALTH = "Mental health conditions"
+        OVERWEIGHT_OR_OBESITY = "Overweight and obesity"
+        PREGNANCY = "Pregnancy"
+        SICKLECELL_OR_THALASSEMIA = "Sickle cell or thalassemia"
+        SMOKING = "Smoking, current or former"
+        TRANSPLANT = "Solid organ or blood stem cell transplant"
+        STROKE_OR_CEREBROVASCULAR = "Stroke or cerebrovascular disease, which affects blood flow to the brain"
+        SUBSTANCE_USE = "Substance use disorders"
+        TUBERCULOSIS = "Tuberculosis"
+        OTHERS = "Others"
+
+    def __init__(self,
+                 firstname: str,
+                 lastname:str,
+                 camp: str,
```

Comment:
Should we change `camp: str` to `camp: Camp` to avoid creating a str object that has no connection with Camp class? Also I think it's be better to have a `InvalidCampException(Exception)` to handle input errors (i.e. when volunteer enter a camp that does not exist).